src/exercise5.js

```diff
@@ -0,0 +1,17 @@
+function dec2bin(num) {
+  return (num >>> 0).toString(2);
+}
+console.log(dec2bin(3));
+
+const fibonnaciSequence = (num)=>{
+  let fibonacci = [0,1];
+  let i = 2;
+  let nextFabonnacci = fibonacci[i-1] + fibonacci[i-2];
+  while(nextFabonnacci < num){
+    fibonacci[i] = nextFabonnacci;
+    i++;
+    nextFabonnacci = fibonacci[i-1] + fibonacci[i-2];
+  }
```

Comment:
on this line `let nextFabonnacci = fibonacci[i-1] + fibonacci[i-2];` 

the value of `fibonacci[i-1]` and `fibonacci[i-2]` are known you ca just replace them to get 
```js
let nextFabonnacci = 1
```
I think you have a typo here `nextFabonnacci `

Since you are using the variable `i` which is acting as an index I would recommend using the `for loop` anyway. 
I think with a for loop you would save some lines of code